layers/+lang/typescript/packages.el

```diff
@@ -89,7 +91,11 @@
                           (progn
                             (with-current-buffer outputbuf (erase-buffer))
                             (write-region nil nil tmpfile)
-                            (if (zerop (apply 'call-process "tsfmt" nil outputbuf nil (list tmpfile)))
+                            (if (zerop (apply 'call-process "tsfmt" nil outputbuf nil
+                                              (list (format
+                                                     "--baseDir='%s' --"
+                                                     default-directory)
```

Comment:
Maybe we can make the same assumption as tide - https://github.com/ananthakumaran/tide/blob/master/tide.el#L102